### PR TITLE
Hibernate Reactive blog

### PR DESCRIPTION
**Pull request by gavinking**

### CODE REVIEW

File: posts/Gavin/2020-12-01-hibernate-reactive.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+So today we're introducing the community to {HR}[Hibernate Reactive], a reactive
+API for Hibernate ORM, which supports non-blocking database clients and reactive
+programming as a paradigm for interacting with the relational database.
+
+Hibernate Reactive reuses most of the implementation of Hibernate ORM, but replaces
+the code which interacts with JDBC with a new layer designed around reactive streams,
+and exposes a new reactive `Session` API to the programmer.
+
+In fact, there's _two_ reactive `Session` APIs, one for programs using Mutiny, and
```
Comment (gunnarmorling) [inline posts/Gavin/2020-12-01-hibernate-reactive.adoc:40]: Would love to see at least one simple example for how the API usage looks like.

Comment (gavinking) [reply posts/Gavin/2020-12-01-hibernate-reactive.adoc:40]: I agree, I was already thinking that. Check staging.

**Comment by gunnarmorling** [comment]
> I agree, I was already thinking that. Check staging.

Yeah, that's nice.

The other thing I was missing a bit is a motivation for _why_ I'd want a
reactive API in the first place. You discuss it towards the end in the
"caution" section, but I felt some sentence at the beginning why this
should excite me would be nice.



>

**Comment by gavinking** [comment]
Well, it's not like i'm really trying to convince anyone to go reactive. We're kinda putting it out there for people who _already_ know they want this.

**Pull request merged**